### DISCUSSION TITLE
Why am I getting this error?

### DISCUSSION THREAD

**Discussion by EvanzGuti10** [flet-dev/flet#2623] [General]
I am new to the world of flet, I was trying to learn with the available documentation, as well as the examples that are included, but when I started I ran into this problem:

Exception in thread Thread-1 (__connection_loop):
Traceback (most recent call last):
  File "C:\Users\Wolf\AppData\Local\Programs\Python\Python312\Lib\threading.py", line 1073, in _bootstrap_inner

How can I solve it, it doesn't let me run any examples, thanks in advance

**Comment by dhelbegor** [comment]
Can you please share the code that you are trying to run so we can see what is going on?

**Reply by EvanzGuti10** [reply]
import flet as ft

def main(page: ft.Page):
    page.title = "Flet counter example"
    page.vertical_alignment = ft.MainAxisAlignment.CENTER

    txt_number = ft.TextField(value="0", text_align=ft.TextAlign.RIGHT, width=100)

    def minus_click(e):
        txt_number.value = str(int(txt_number.value) - 1)
        page.update()

    def plus_click(e):
        txt_number.value = str(int(txt_number.value) + 1)
        page.update()

    page.add(
        ft.Row(
            [
                ft.IconButton(ft.icons.REMOVE, on_click=minus_click),
                txt_number,
                ft.IconButton(ft.icons.ADD, on_click=plus_click),
            ],
            alignment=ft.MainAxisAlignment.CENTER,
        )
    )

ft.app(target=main)

**Comment by ndonkoHenri** [comment]
I tried your code and had no issue. 
If you are still facing this error, can you provide more details? (full error traceback, python version, flet version...)
